public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -135,28 +134,28 @@ export default class Dashboard extends Component {
         alertState,
         monitorIds,
       };
-      const queryParamsString = queryString.stringify(params);
+
+      const { httpClient, history, notifications, location, perAlertView } = this.props;
       location.search;
```

Comment:
Richard and I discussed this line offline. It looks like it was added in this [migration PR](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/commit/58172d9e0709e5556cb8d5cb22af1577de0717a0#diff-90a4d66c6d6bea36116eea37f5ab884f0bb10fce8f7bc1997fc7c24cb01a9f05R183) back in the ODFE repo. It did not exist on the [previous branch](https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/opendistro-1.9/public/pages/Dashboard/containers/Dashboard.js#L184). The `location.search` value in this context would just be a `string`; so this line didn't seem to accomplish anything in this function as its value wasn't even assigned to a local variable.  It seems safe to remove this line.